File: Tema 5 - Arrrays/ejercicio3/archives/automaticamente/automaticamente.py
"""
Este módulo muestra un MENSAJE incrementando el valor de i.
"""
import sys
import json

if len(sys.argv) > 1:
    NUMERO = json.loads(sys.argv[1])
    MENSAJE = ""
    MENSAJE += (
        "<table border='1'><tr><th>Indice</th><th>Numero</th><th>Rotado</th></tr>"
    )
    VOLTEADO = NUMERO.copy()
    AUX = VOLTEADO.pop()
    VOLTEADO.insert(0, AUX)
    for contador, valor_numero in enumerate(NUMERO):
        DESCRIPCION = valor_numero
        ROTAR = VOLTEADO[contador]
        MENSAJE += f"<tr><td>{contador}</td><td>{DESCRIPCION}</td><td>{ROTAR}</td></tr>"

    MENSAJE += "</table>"

    print(f"{MENSAJE}")

File: Tema 5 - Arrrays/ejercicio5/archives/automaticamente/automaticamente.php
<?php
if (isset($_GET["numerosAuto"])) {
  $numero = json_decode($_GET["numerosAuto"]);
  $mensaje = "<table border='1'><tr><th>Indice</th><th>Numero</th><th>Rotado</th></tr>";

  $volteado = $numero;
  $aux = array_pop($volteado);
  array_unshift($volteado, $aux);
  foreach ($numero as $indice => $elemento) {
    $descripcion = $elemento;
    $rotar = $volteado[$indice];
    $mensaje .= "<tr><td>$indice</td><td>$descripcion</td><td>$rotar</td></tr>";
  }

  $mensaje .= "</table>";
  echo $mensaje;
}
